src/forecast.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";

```

Comment:
Great implementation of the table, can remove the react import
consider using mui or bootstrap for easy styling and responsive sizes.